src/pages/community.js

```diff
@@ -0,0 +1,683 @@
+        title="Community home"
+        description="Community homepage description"
+      />
+      <PageHero isReverse content={heroContent} />
+      <Divider />
+      <StyledPurpleContainer>
+        <Content>
+          <CentralColumn>
+            <H2>Why get involved?</H2>
+          </CentralColumn>
+          <CardContainer>
+            {whyGetInvolvedCards.map((card, idx) => (
+              <StyledGetInvolvedCard
+                key={idx}
+                emoji={card.emoji}
+                title={card.title}
+                description={card.description}
+              />
+            ))}
+          </CardContainer>
+        </Content>
+      </StyledPurpleContainer>
+      <StyledGrayContainer>
+        <StyledContent>
+          <IntroRow>
+            <IntroLeftColumn>
+              <H2 id="get-involved">
+                {/* <Translation id="page-community-get-involved" /> */}
+                How can I get involved?
+              </H2>
+              <Subtitle>
+                {/* <Translation id="page-community-get-involved-description" /> */}
+                There are many ways to get involved in the fast-growing Ethereum
+                community; you can join one of the popular online communities,
+                attend an event, join a meetup group, contribute to a project,
+                or participate in one of the many online forums about Ethereum.
+              </Subtitle>
+              <ButtonLink to="/community/grants">
+                {/* <Translation id="page-community-get-involved-button" /> */}
+                Get involved!
+              </ButtonLink>
+            </IntroLeftColumn>
+            <ImageContainer>
+              <IntroImage
+                fluid={data.developerBlocks.childImageSharp.fluid}
+                alt={translateMessageId(
+                  "page-community-get-involved-image-alt",
+                  intl
+                )}
+              />
+            </ImageContainer>
+          </IntroRow>
+          <StyledCardContainer>
+            {cards.map((card, idx) => (
+              <StyledCard
+                key={idx}
+                title={card.title}
+                description={card.description}
+                to={card.to}
+                image={card.image}
+                alt={card.alt}
+              />
+            ))}
+          </StyledCardContainer>
+        </StyledContent>
+      </StyledGrayContainer>
+      <OpenSourceContainer>
+        <RowReverse>
+          <FeatureContent>
+            {/* <StyledH2>
+              <Translation id="page-index-what-is-ethereum" />
+            </StyledH2> */}
+            <H2>
+              {/* <Translation id="page-community-open-source" /> */}
+              Creator? Builder? Get paid for your work.
+            </H2>
+            <Subtitle>
+              {/* <Translation id="page-community-open-source-description" /> */}
+              Are you building on Ethereum, or do you want to? Funding grants
```

Comment:
I like this section - maybe also mentions there's companies hiring for thousands of jobs? i.e. make it more broad about working in the space vs. just grants?